src/Serilog.Sinks.Couchbase/LoggerConfigurationCouchBaseExtensions.cs

```diff
@@ -38,22 +41,23 @@ public static class LoggerConfigurationCouchbaseExtensions
         /// <exception cref="ArgumentNullException">A required parameter is null.</exception>
         public static LoggerConfiguration Couchbase(
             this LoggerSinkConfiguration loggerConfiguration,
-            string[] couchbaseUriList, 
+            List<Uri> couchbaseUriList, 
```

Comment:
If we change the parameter type here, `IEnumerable<Uri>` is possibly a better option since we enumerate the argument immediately?